src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -66,6 +70,66 @@ const Row = ({ children, Icon, label }) => {
     </>
   );
 };
+
+const DoneButton = styled(Button)`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`;
+const TextRowEditable = ({ disabled, label, onClick, text }) => {
+  const [value, setValue] = useState(text);
+  const [isEdit, setIsEdit] = useState(false);
+
+  function handleChange(event) {
+    setValue(event.target.value);
+  }
+  //disabled have higher priority
+  let isEditable = isEdit;
```

Comment:
Could we do something similar to how you handled line 95?

It's just odd to reassign variables like this especially when they're already tied to a state

Reply:
yeah, you are right